core-api/src/main/java/com/sequenceiq/distrox/api/v1/distrox/endpoint/DistroXV1Endpoint.java

```diff
@@ -105,6 +106,12 @@ StackViewV4Responses list(
             nickname = "postDistroXV1")
     StackV4Response post(@Valid DistroXV1Request request);
 
+    @POST
+    @Path("/operational_database")
```

Comment:
I would not put OPDB into the path or method name, since this can be used for internal distrox creation also

Reply:
It could even be it's own Controller, who know what else needs to be duplicated.
We should have some form of pattern for defining internal API's.

We already have examples like Flow and CB legacy API.

Reply:
I reused the other method call. It's easy to move this into a new controller later if we need to.